server/routers/analysis.py

```
@@ -0,0 +1,276 @@
+        "total_incidents": total_incidents,
+        "high_priority": high_priority,
+        "response_rate": 100 if total_incidents > 0 else 0,
+        "avg_response_time": 10,
+        "tweets_recognized": total_posts,
+        "prediction_accuracy": 100 if accurate_predictions > 0 else 0,
+        "anomalies_detected": anomalies,
+    }
+
+
+@router.get("/crisis-trends")
+def get_crisis_trends(days: int = 30, db: Session = Depends(get_db)):
```

Ruff (0.14.0)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)